docs/conf.py

```diff
@@ -41,8 +41,8 @@
 # source_suffix = ['.rst', '.md']
 source_suffix = ".rst"
 
-# The master toctree document.
-master_doc = "index"
+# The main toctree document.
+main_doc = "index"
```

Comment:
I am not sure if this is something that Sphinx expects to have a particular name; have you tried building the docs to various targets (at least PDF and HTML I guess?) with this change in?

Reply:
Yeah, you're probably right. I built them to HTML successfully, but since it's in the Sphinx docs (https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=master_doc#confval-master_doc) we probably shouldn't mess with it.